washington_a04b.js

```diff
@@ -0,0 +1,27 @@
+/*
+Timothy Washington
+washington_a04b.js
+INFO 2124 - Javascript I
+Thoendel
+Creation date: Jan 12, 2020
+*/
+//comment Legend: {c=creates variable, s=sets a variable to a value}
+const name="Timothy";//cs
+const wei=1.1;//cs
+const hei=65;//cs
+const age=20;//cs
+const marsGravity=0.38;//cs
+const inchesToCentimeters=2.54;//cs and removed unecessary naming confusion
+var cAge=parseInt(String(age*365));//cs
```

Comment:
this seems like a lot of unnecessary conversion.  Also - use of var is deprecated - we should be using let.